rest-api/dao/participant_summary_dao.py

```diff
@@ -336,11 +336,15 @@ def to_client_json(self, model):
     return result
 
   def _decode_token(self, query_def, fields):
+    """ If token exists in participant_summary api, decode and use lastModified to add a buffer
+    of 60 seconds. This ensures when a _sync link is used no one is missed. This may return
+    participants more than once if it is called to often, which should be handled on the client
```

Comment:
I would tweak this: we fully expect that at least one participant (the last one returned) will be repeated in the next sync response, along with any participants modified in the previous 60 seconds. (This will happen whether the client polls every 5 minutes or every hour.)